Automattic/course
# Add style variations

This PR adds three styles variations to the theme:
- Blue
- Dark
- Gold

## Testing Instructions
- Add or re-add the mailing list and testimonial patterns to a page.
- Go to the site editor.
- Open the Styles sidebar.
- Click _Browse styles_.
- Select the _Blue_ variation.
- Check that the theme updates to the new colors and styles. Check posts, pages, patterns, 404 page, search page and archive page to ensure they look good.
- Repeat the above steps for the _Dark_ and _Gold_ variations.

## Screenshots

### Blue

![screencapture-senseitheme-local-2022-11-17-14_58_21](https://user-images.githubusercontent.com/1190420/202546641-248b5ff0-6b0e-47b2-98b8-4b7c279052de.png)

### Dark

![screencapture-senseitheme-local-2022-11-17-15_05_14](https://user-images.githubusercontent.com/1190420/202548185-814d639d-48b6-495b-9541-3570c0c62e7f.png)

### Gold

![screencapture-senseitheme-local-2022-11-17-15_00_12 (1)](https://user-images.githubusercontent.com/1190420/202547017-c5416e8d-65ae-4348-90dc-38ae4ae9ce86.png)
